db/migrate/20170711235422_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+ROM::SQL.migration do
+  change do
+    create_table :articles do
+      primary_key :id
```

Comment:
@JuanitoFatas certainly a valid thing to consider – probably worth bringing up in the rom-rb or sequel projects? For this application here my plan is to just follow the typical approaches for the various gems we use.